react/components/Textarea/index.js

```diff
@@ -163,6 +182,8 @@ Textarea.propTypes = {
   onFocus: PropTypes.func,
   /** onBlur event */
   onBlur: PropTypes.func,
+  /** Character countdown at the bottom right */
+  characterCountdown: PropTypes.bool,
```

Comment:
O que acha de mostrar o contador no momento que tiver `maxLength` setado, ao invés de ter essa prop?

Mas se for manter essa prop, a descrição dela tem que mencionar que leva em conta o `maxLength` pra fazer a contagem.

Reply:
Eu cheguei a pensar nisso, mas eu não queria correr o risco de afetar alguém que vinha usando maxLength antes e de repente passaria a ter o contador lá. Acho que é bom deixar como opcional, e ter uma prop específica pra isso garante que ninguém vai ganhar um contador sem querer.

Reply:
@jullyanav me metendo aqui na discussão: acho que não estamos num momento de projeto de se preocupar nesse nível com retrocompatibilidade, o mais importante é fazer o que acreditamos ser o certo. Todos que estão usando o Styleguide estão avisados de que é um projeto em construção.
E pra falar a verdade duvido que alguém se incomode com um contador amigo aparecendo se a pessoa já tava usando maxlength!